styles.css

```diff
@@ -539,7 +560,8 @@ footer hr {
 /* RESPONSIVENESS */
 
 /* mobile landscape */
-@media screen and (min-width: 675px) and (max-width: 939px) {
+
+@media screen and (min-width: 675px) and (max-width: 767px) {
```

Comment:
- [Optional] I would advise you to use `min-width: 768px ` or above that's the requirement. YOu can use the below. (PLease implement it)
` @media screen and (min-width: 768px)` there is no need for the max-width added.

###### Happy coding :blush: :+1: 
- Feel free to reach out to me on slack `@kanu mike chibundu`